homeworks-05-2022/src/main/java/io/codelex/secondtest/exercise3/Joiner.java

```diff
@@ -0,0 +1,19 @@
+package io.codelex.secondtest.exercise3;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Joiner<T> {
+
+    String seperator;
+
+    public Joiner(String seperator) {
+        this.seperator = seperator;
+    }
+
+    public String join(List<T> listToJoin, Integer count) {
```

Comment:
Interesanti no kurienes te uzradās tas Integer count? :D Jo faktiski viņš īsti nebija prasīts.

Reply:
Mani tādā gadījumā samulsināja šis teksts -> [... allows you to specify any number of T-type objects. ...]